### PR TITLE
Adds documentation for the proposed AirVisual sensor platform

### PR DESCRIPTION
**Pull request by bachya**
**Description:**
Adds documentation for the proposed AirVisual sensor platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/9320

### CODE REVIEW

File: source/_components/sensor.airvisual.markdown [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+logo: airvisual.jpg
+ha_category: Health
+ha_release: 0.53
+ha_iot_class: "Cloud Polling"
+---
+
+The `airvisual` sensor platform queries the [AirVisual](https://airvisual.com/) API for air quality
+data on the nearest city to a latitude and longitude. The resulting information
+creates sensors for the Air Quality Index (AQI), the human-friendly air quality
+level, and the main pollutant of that area. Sensors that confirm to either/both
```
Comment (frenck) [inline source/_components/sensor.airvisual.markdown]: **Spelling & Grammar**:
- `confirm` -> `conform`

Comment (bachya) [reply source/_components/sensor.airvisual.markdown]: Fixed – thank you!

**Comment by frenck** [comment]
Thanks, @bachya. Can be merged as soon as the parent PR is merged.

**Comment by bachya** [comment]
Thank you!

**Pull request merged**